src/location-input/LocationInput.svelte

```diff
@@ -90,14 +90,29 @@
       );
     }
   };
+jQuery(document).on('click', ".input-address-container", function() {
```

Comment:
```suggestion
jQuery(".input-address-container").on('click', function() {
```
Can we use this syntax instead? Idk if jQuery is smart enough to not put the listener on the document, but if not - this could be a performance bottleneck.